tests/system/python/pair/test_pyton_north_service_pair.py

```diff
@@ -21,7 +21,7 @@
 
 # Local machine
 local_south_plugin = "sinusoid"
-local_south_asset_name = "sinusoid"
+local_south_asset_name = "python-north-service-pair"
```

Comment:
Why should this be the same? As remote north?

Reply:
Remote asset name does not matter. the local set up sends data to remote by http north and then to PI. So if the local asset name is for example asset1 then pi point created will be asset1.